metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    location = models.IntegerField()
```

Comment:
This is supposed to be the character index of where the highlighted sentence starts? I'm not sure exactly if that will work well as we're embedding an iframe of some HTML, so we might need some more info, i.e. the ID of the container of the highlighted text, etc.